docs/index_es.html

```diff
@@ -0,0 +1,118 @@
+          <li>No se sorprende cuando alguien no sabe algo.</li>
+          <li>Está dispuesto/a y es capaz de colaborar con otros.</li>
+          <li>Celebra públicamente a otro/as por sus victorias.</li>
+          <li>Hace preguntas antes de proporcionar comentarios críticos.</li>
+          <li>Da retroalimentación dura en privado.</li>
+          <li>Trata a otras personas como a <em>el/ella</em> le gustaría ser tratado/a.</li>
+          <li>Proporciona revisiones de código y retroalimentación a sus compañero/as que son constructivos, útiles y presentados con tacto, ayudando a sus compañero/as a crecer personal y profesionalmente.</li>
+          <li>Expresa su aprecio por las revisiones de código y las retroalimentaciones de sus compañero/as que son constructivos y útiles.</li>
+          <li>A veces se siente herido/a por la retroalimentación crítica, pero no reacciona destructivamente.</li>
+          <li>A veces pasa horas depurando un problema menor y lo soluciona en unos minutos.</li>
+          <li>A veces toma breves descansos para despejarse la cabeza.</li>
+          <li>A veces se siente frustrado/a por su entorno y necesita desahogarse.</li>
+          <li>Comete errores de vez en cuando y encuentra crecimiento en esos errores.</li>
+          <li>Están dispuesto/as a admitir cuando están equivocado/as y no tienen miedo de decir "No lo sé".</li>
+          <li>Puede que le guste o no escribir documentación, pero lo hace de todos modos para futuro/as mantenedore/as.</li>
+          <li>Puede que le guste o no escribir pruebas, pero trata de aprender a hacerlo si el equipo o proyecto lo necesita.</li>
+          <li>Agradece a lo/as demás por su tiempo, esfuerzo y energía.</li>
+          <li>Puede tener fondos de escritorio coloridos.</li>
+          <li>Apoya código en producción, incluso si no lo escribieron.</li>
+          <li>Puede sentirse como un/a impostor/a a veces, y entiende que otro/as también pueden hacerlo.</li>
+          <li>Cree que todo/as en la sala son tan inteligentes y capaces como ello/as.</li>
+          <li>Ayudará a subir de nivel a otro/as y pedirá ayuda cuando la necesiten.</li>
+          <li>Nunca deja de aprender, pero puede sentirse totalmente abrumado/a por la cantidad de aprendizaje que hay que hacer.</li>
+          <li>Trata de mantener las discusiones productivas y permite que otro/as expresen su opinión antes de que el equipo tome una decisión.</li>
+          <li>Está dispuesto/a a abandonar su zona de confort..</li>
+          <li>Está dispuesto a dejar sus contribuciones a la comunidad a su manera cuando es posible y aprecia las formas en que otro/as contribuyen cuando pueden sentirse cómodo/as..</li>
+          <li>Puede ser un/a programador/a lento/a.</li>
+          <li>Tiene días productivos e improductivos.</li>
+          <li>No se toma demasiado en serio.</li>
+          <li>Dice: "Nunca he oído hablar de eso", en lugar de asentir y fingir.</li>
+          <li>es confiable.</li>
```

Comment:
¡La traducción está muy bien! Algo pequeñito:

```suggestion
          <li>Es confiable.</li>
```